mod/wikirate/spec/set/type/claim_spec.rb

```diff
@@ -102,20 +102,23 @@ def create_page iUrl=nil
       context "when there is no topic " do
         it "shows tip about adding topic" do
           claim_card = Card.create :type_id=>Card::ClaimID, :name=>@claim_name ,:subcards=>{ '+company'=>'apple','+source' => {:content=>"[[#{@sourcepage.name}]]",:type_id=>Card::PointerID}}
-          expect(claim_card.format.render_tip).to include('improve this claim by adding a topic.')
+          expected_line = 'improve this note by adding a topic.'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.